### ISSUE TITLE
sshchat support is broken

### ISSUE THREAD

**Issue by ValdikSS** [42wim/matterbridge#665]
**Describe the bug**
When configuring sshchat bridge, it's either does not relay messages or cause matterbridge panic.

**To Reproduce**
Configure as follows:
```
[sshchat.chat]
Server="chat.example.com"
Nick="bot"
RemoteNickFormat="<{NICK}> "

[[gateway]]
name="chat"
enable=true

[[gateway.inout]]
account="telegram.otherbridge"
channel="-123456789"

[[gateway.inout]]
account="sshchat.chat"
```

**Expected behavior**
Relaying works

**Screenshots/debug logs**
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0xa64658]

goroutine 32 [running]:
github.com/42wim/matterbridge/bridge/sshchat.(*Bsshchat).Send(0xc00041a220, 0xc0007d4450, 0x30, 0x0, 0x0, 0xc000416b10, 0x9, 0xc00054b153, 0x9, 0x0, ...)
	/go/src/github.com/42wim/matterbridge/bridge/sshchat/sshchat.go:81 +0x6a8
github.com/42wim/matterbridge/gateway.(*Gateway).handleMessage(0xc000267da0, 0xc0007d4450, 0x30, 0x0, 0x0, 0xc000416b10, 0x9, 0xc00054b153, 0x9, 0x0, ...)
	/go/src/github.com/42wim/matterbridge/gateway/gateway.go:330 +0x8db
github.com/42wim/matterbridge/gateway.(*Router).handleReceive(0xc0002b37a0)
	/go/src/github.com/42wim/matterbridge/gateway/router.go:109 +0x298
created by github.com/42wim/matterbridge/gateway.(*Router).Start
	/go/src/github.com/42wim/matterbridge/gateway/router.go:64 +0x571
```

**Environment (please complete the following information):**
Linux x86_64, version: 1.12.3 35736bb

**Comment by ValdikSS** [comment]
By the way, there's no example for sshchat in matterbridge.sample. Have I configured it correctly?

**Comment by ValdikSS** [comment]
Fixed in #661
